### PR TITLE
Adding ability to scale memory and disk.  Also adding restart command…

### LINKED ISSUE #10: Add scaling memory & scaling disk  and restarting app capabilities
Currently we only allow the scaling of instances `hubot app scale [app] [num] - Scale app to a number of instances.`

```
NAME:
   scale - Change or view the instance count, disk space limit, and memory limit for an app

USAGE:
   cf scale APP_NAME [-i INSTANCES] [-k DISK] [-m MEMORY] [-f]

OPTIONS:
   -f      Force restart of app without prompt
   -i      Number of instances
   -k      Disk limit (e.g. 256M, 1024M, 1G)
   -m      Memory limit (e.g. 256M, 1024M, 1G)
```

cf scale allows you do the above we need to provide more options.

### PR DESCRIPTION
**Pull request by JonHoughton**
….  Closes #10.

### CODE REVIEW

File: src/scripts/app.restart.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	// Add more languages to the list of locales when the files are created.
+	directory: __dirname + '/../messages',
+	defaultLocale: 'en',
+	// Prevent messages file from being overwritten in error conditions (like poor JSON).
+	updateFiles: false
+});
+// At some point we need to toggle this setting based on some user input.
+i18n.setLocale('en');
+
+const STOP = /(app\srestart)\s(.*)/i;
```
Comment (chambrid) [inline src/scripts/app.restart.js]: Probably should use a different constant.

File: test/bluemix.apps.test.js [hunk 2]
```diff
@@ -36,6 +36,19 @@ const invalidApp = 'testApp4';
...
+		}
+	}).then(() => false).catch(() => true).then((success) => {
+		// Great.  Move on to tests
+		expect(room.messages.length).to.eql(len);
+	});
+}
+
 // Passing arrow functions to mocha is discouraged: https://mochajs.org/#arrow-functions
 // return promises from mocha tests rather than calling done() - http://tobyho.com/2015/12/16/mocha-with-promises/
 describe('Interacting with Bluemix via Slack', function() {
```
Comment (chambrid) [inline test/bluemix.apps.test.js]: Change to Interacting with RegEx

Comment (JonHoughton) [inline test/bluemix.apps.test.js]: Done.

File: src/scripts/app.restart.js (hunk 1 above)
Comment (JonHoughton) [inline src/scripts/app.restart.js]: Done.

**Comment by JonHoughton** [comment]
@chambrid Changes have been made per review comments.  Ready to merge.

**Comment by chambrid** [comment]
👍

**Pull request merged**